include/osm2rdf/ttl/Constants.h

```diff
@@ -72,6 +71,8 @@ inline std::string IRI__OSM2RDF_GEOM__OBB;
 inline std::string IRI__OSM2RDF_MEMBER__ID;
 inline std::string IRI__OSM2RDF_MEMBER__ROLE;
 inline std::string IRI__OSM2RDF_MEMBER__POS;
+inline std::string IRI__OSM2RDF_WAYNODE__POS;
+inline std::string IRI__OSM2RDF_WAYNODE__ID;
```

Comment:
Given the new prefixed names for these, the variable name is no longer appropriate. Please also check the other variable names, it would be great if they were all consistent.

Reply:
I made a pass over the variable names, in particular the prefixes should now match the ones mentioned in the variable name. I now also consistently use `__` as a separator, and `_` as a whitespace indicator.